### DISCUSSION TITLE
I'm Stuck

### DISCUSSION THREAD

**Discussion by brendonc75** [ParadoxAlarmInterface/pai#395] [Q&A] [answered]
Hellos Everyone.

For the Last 2 Weeks I've to trying and trying to get this right, either I'm just clueless or very lost ( SAME SAME at this point)

Ive followed every part of the video and read everything,

below i have copied the yaml code from Pai 

can someone have a look and let me know what im doing wrong

MG5050   V: 7.01.012
Ip150        V : 5.02.058
![Screenshot 2023-09-22 075411](https://github.com/ParadoxAlarmInterface/pai/assets/51416090/e6fae2f8-b125-480f-aff6-1b157e63491d)

LOGGING_LEVEL_CONSOLE: 20
LOGGING_LEVEL_FILE: 40
CONNECTION_TYPE: IP
SERIAL_PORT: /dev/ttyUSB0
SERIAL_BAUD: 9600
IP_CONNECTION_HOST: 192.168.0.250
IP_CONNECTION_PORT: 10000
IP_CONNECTION_PASSWORD: "paradox"
KEEP_ALIVE_INTERVAL: 10
LIMITS:
  zone: auto
  user: 1-10
  door: ""
  pgm: 1-5
  partition: auto
  module: ""
  repeater: ""
  keypad: ""
  key-switch: ""
SYNC_TIME: true
SYNC_TIME_MIN_DRIFT: 120
PASSWORD: "0000"
MQTT_ENABLE: true
MQTT_HOST: core-mosquitto
MQTT_PORT: 1883
MQTT_KEEPALIVE: 60
MQTT_USERNAME: Pai
MQTT_PASSWORD: Pai
MQTT_HOMEASSISTANT_AUTODISCOVERY_ENABLE: false
COMMAND_ALIAS:
  arm: partition all arm
  disarm: partition all disarm
MQTT_COMMAND_ALIAS:
  armed_home: arm_stay
  armed_night: arm_sleep
  armed_away: arm
  disarmed: disarm
HOMEASSISTANT_NOTIFICATIONS_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
HOMEASSISTANT_PUBLISH_PARTITION_PROPERTIES:
  - target_state
  - current_state
HOMEASSISTANT_PUBLISH_ZONE_PROPERTIES:
  - open
  - tamper
PUSHBULLET_CONTACTS: []
PUSHBULLET_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
PUSHOVER_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
PUSHOVER_BROADCAST_KEYS: []
SIGNAL_CONTACTS: []
SIGNAL_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper

GSM_CONTACTS: []
GSM_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
IP_INTERFACE_ENABLE: false
IP_INTERFACE_PASSWORD: paradox
DUMMY_EVENT_FILTERS: []
![Screenshot 2023-09-22 080022](https://github.com/ParadoxAlarmInterface/pai/assets/51416090/8234c323-bb06-4aef-829e-31b56c6c419c)

**Comment by yozik04** [comment]
According to the provided log your IP150 does not respond.

**Reply by brendonc75** [reply]
is there something i'm missing in-order  to mark it respond

**Reply by yozik04** [reply]
try
$ telnet 192.168.0.250 10000
Will it bring up the connection? If not then it is not PAI to blame.

**Reply by brendonc75** [reply]
ok i will check, if that is the case what can i do?

**Reply by brendonc75** [reply]
Afternoon All, so i managed to get connect to my panel, i have a port error on the Ip150 Module, however i now have an authentication Error (  Invalid Pc Password, Default is " 0000 " 
![Screenshot 2023-09-26 135029](https://github.com/ParadoxAlarmInterface/pai/assets/51416090/7b4e48ab-4c9f-4bbf-a4a7-b74d73a36d48)

**Reply by yozik04** [reply, marked as answer]
Try 'aaaa'

**Reply by brendonc75** [reply]
100% thank you very very much.. after 5 months of locking myself out every time.
thank you very much once again

**Reply by yozik04** [reply]
These MG models some times want 'a' instead of '0'. No idea why. I could not figure out a pattern.
